plugins/color_scheme_dev.py

```diff
@@ -256,16 +256,33 @@ def run(self):
                 (setting, self.get_scheme_path(view, setting))
                 for setting in ('dark_color_scheme', 'light_color_scheme')
             ]
+            current_os_mode = sublime.ui_info()['system']['style']
             choices = [
                 sublime.QuickPanelItem(setting, details=str(path), kind=KIND_SCHEME)
                 for setting, path in paths
             ]
 
+            for idx, choice in enumerate(choices):
+                if current_os_mode == 'dark' and choice.trigger == 'dark_color_scheme':
+                    choice.annotation = 'Active'
+                    selected_index = idx
+                elif current_os_mode == 'light' and choice.trigger == 'light_color_scheme':
+                    choice.annotation = 'Active'
+                    selected_index = idx
+                else:
+                    choice.annotation = ''
+                    selected_index = -1
```

Comment:
This `else` branch will result in the selected index being reset if the first case (dark) matches, after which the light case cannot match on the second iteration.

This would work better as follows:

```suggestion
            selected_index = -1
            for idx, choice in enumerate(choices):
                if choice.trigger.startswith(current_os_mode):
                    choice.annotation = 'Active'
                    selected_index = idx
```

Reply:
Thanks. Your solution is much cleaner. I have a lot to learn.